src/main/kotlin/racingcar/Application.kt

```diff
@@ -0,0 +1,15 @@
+package racingcar
+
+import racingcar.controller.World
+
+fun main() {
+    try {
+        World()
+    } catch (e: Exception) {
+        when (e) {
+            is IllegalArgumentException, is IllegalStateException -> {
+                println("[ERROR]: " + e.message)
+            }
+        }
+    }
```

Comment:
1. 프로그램 내에서 `IllegalStateException`를 throw하는 곳이 없기에 `IllegalArgumentException`만 헨들링해도 될 것 같습니다.
```suggestion
    try {
        World()
    } catch (e: IllegalArgumentException) {
        println("[ERROR]: " + e.message)
    }
```
2. `catch (e: Exception)`문으로 모든 에러를 catch하고 있는데요, 이러한 코드는 프로그램이 커질수록 예상치 못한 버그를 발견하기 어렵게 만들 수 있어요. (예를 들어 에러가 발생했는데 해당 에러를 catch문이 먹어서 프로그래머가 인지하지 못하는 일이 생길 수 있습니다) 나중에 미션을 진행하면서 조금이라도 구체적인 catch문을 작성하여 이러한 문제를 예방하는 편이 좋습니다.

Reply:
안녕하세요 코치님! 😊 제 첫 리뷰는 레아 코치님이셨군요 ! 잘부탁드립니다. 

1. catch( )를 사용한 코드를 삭제한 후 저 부분을 다시 체크하지 못했던 것 같습니다. 😂
   앞으로는 코드 수정 후 이런 부분도 확인해야할 것 같네용..

2. 원래는 
```kotlin
try{
    ...
} catch (e: IllegalArgumentException) {
    ...
} catch (e: IllegalStateException) {
    ...
}
```
이런식으로 길게 늘이는 것이 싫어 코드를 위와 같이 작성했었는데 catch문이 에러를 먹어버릴 수도 있겠다는 생각을 못했었네요! 이 부분은 코치님이 설명해주신대로 정정하겠습니다!